torch/nn/_functions/thnn/im2col.py

```diff
@@ -0,0 +1,47 @@
+from torch.autograd.function import Function
+from torch._thnn import type2backend
+
+from . import _all_functions
+
+class Im2Col(Function):
```

Comment:
This is an legacy style of writing Functions. See our extending autograd docs to see how new style functions look like

Reply:
Updated to new style